.github/workflows/snyk.yml

```diff
@@ -0,0 +1,63 @@
+name: "Snyk"
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  DEFAULT_JDK_VERSION: 11
+
+jobs:
+  quarkus:
+    name: Quarkus
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: ${{ env.DEFAULT_JDK_VERSION }}
+          cache: maven
+      
+      - name: Build Keycloak
+        run: mvn clean install -nsu -B -e -Pdistribution -Dmaven.test.skip -DskipQuarkus -DskipTestsuite -DskipExamples -DskipTests
+
+      - uses: snyk/actions/setup@master
+      - name: Check for vulnerabilities
+        run: snyk test --policy-path=.snyk --all-projects --prune-repeated-subdependencies --exclude=tests --sarif-file-output=quarkus-report.sarif quarkus
+        continue-on-error: true
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+
+      - name: Upload scanner results to GitHub
+        uses: github/codeql-action/upload-sarif@v1
+        with:
+          sarif_file: quarkus-report.sarif
+
+  operator:
+    name: Operator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: ${{ env.DEFAULT_JDK_VERSION }}
```

Comment:
Not entirely sure but I believe you need to specify a `distribution` property here.
```suggestion
          java-version: ${{ env.DEFAULT_JDK_VERSION }}
          distribution: zulu
```